src/backend/launcher.ts

```diff
@@ -372,26 +374,38 @@ async function prepareWineLaunch(
     }
   }
 
-  if (runner === 'gog' && experimentalFeatures?.cometSupport !== false) {
-    if (isOnline() && !(await isInstalled('comet_dummy_service'))) {
-      await download('comet_dummy_service')
-    }
-    const installerScript = join(
-      runtimePath,
-      'comet_dummy_service',
-      'install-dummy-service.bat'
-    )
-    if (existsSync(installerScript)) {
-      await runWineCommand({
-        commandParts: [installerScript],
-        gameSettings,
-        protonVerb: 'runinprefix'
-      })
-    } else {
-      logWarning(
-        "Comet dummy service isn't downloaded, online functionality may not work"
+  try {
+    if (runner === 'gog' && experimentalFeatures?.cometSupport !== false) {
+      const communicationSource = join(
+        publicDir,
+        'bin/x64/win32/GalaxyCommunication.exe'
       )
+
+      const winePrefix = gameSettings.winePrefix
+      let driveLocation =
+        'drive_c/ProgramData/GOG.com/Galaxy/redists/GalaxyCommunication.exe'
+      if (gameSettings.wineVersion.type === 'proton') {
+        driveLocation = join('pfx', driveLocation)
+      }
+      const communicationDest = join(winePrefix, driveLocation)
```

Comment:
Maybe this can be replaced with the `getWinePath` method? https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/17fd9fbf35c37d55daafa3fc3021db19f3c93dd7/src/backend/launcher.ts#L1326

not a blocker though